### PR TITLE
doc: add android 10.0.0 release blog post

### PR DESCRIPTION
**Pull request by erisu**
### Motivation and Context

Cordova Android 10.0.0 Release Blog Post

### CODE REVIEW

File: www/_posts/2021-07-20-cordova-android-10.0.0.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+author:
+    name: Bryan Ellis
+title:  "Cordova Android 10.0.0 Released!"
+categories: announcements
+tags: news releases
+---
+
+We are happy to announce that we have just released `Cordova Android 10.0.0`!  This is one of Cordova's supported platforms for building Android mobile applications.
+
+* [cordova-android@9.1.0](https://www.npmjs.com/package/cordova-android)
```
Comment (dpogue) [inline www/_posts/2021-07-20-cordova-android-10.0.0.md:12]: `10.0.0` :trollface:

File: www/_posts/2021-07-20-cordova-android-10.0.0.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+    * Android Gradle Plugin (AGP): `4.2.2`
+    * Google Services Gradle Plugin: `4.3.5`
+
+* **Node Support**
+
+    Since Node 10 is no longer being supported by the Node.js team, we have dropped support for Node 10.
+
+* **Java 11 Support**
+
+    With the current release of Android Studio 4.2, the Android tooling can now support running on JAVA 11. Please note that this does not mean you can start compiling JAVA 11 source code.
```
Comment (dpogue) [inline www/_posts/2021-07-20-cordova-android-10.0.0.md:72]: Just use "Java" not "JAVA" here?

Comment (erisu) [reply www/_posts/2021-07-20-cordova-android-10.0.0.md]: Fixed

File: www/_posts/2021-07-20-cordova-android-10.0.0.md (hunk 1 above)
Comment (erisu) [reply www/_posts/2021-07-20-cordova-android-10.0.0.md]: :trollface:

**Review by dpogue — APPROVED** [review]

**Pull request merged**